templates/zerver/help/change-your-language.md

```diff
@@ -1,37 +1,42 @@
 # Change your language
 
-By default, the Zulip UI is displayed in English.  However, Zulip has
-full support for internationalization, and the UI has been translated
-into many other languages.  You can see the Zulip UI in other
-languages temporarily by modifying the URLs using the appropriate
-language code, e.g. `/ru/` for Russian.
+By default, the Zulip UI is displayed in English.  However, Zulip has full
+support for internationalization. You can see which languages Zulip supports,
+and help add support for new languages by visiting
+**[Transifex](https://www.transifex.com/zulip/zulip/)**.
 
-Additionally, you can change the default language in the Zulip UI for
-your Zulip account using the following procedure.
-
-!!! warn ""
-    **Warning**: Because for languages, only some of the strings in
-    the Zulip UI have been translated, you may still see English words
-    interspersed throughout Zulip.
+You can change your personal default language for Zulip using the following
+procedure.
 
 {settings_tab|display-settings}
 
-2. Next to the **Default language** heading, select the **[Change]** link.
+2. Under the **Language Settings** heading, select the language next to the
```

Comment:
These should be in the same style as the macro. So "Under X, select .." type of thing, rather than "Under the X heading, select .."